specs/elements/TextBlock.md

```diff
@@ -103,6 +103,7 @@ Describes what to do when an unknown element is encountered or the requires of t
 * **Required**: No
 * **Allowed values**:
   * `ActionSet`
+  * `Column`
```

Comment:
>Column [](start = 5, length = 6)

These changes now indicate that a non-column type (like TextBlock) can fallback to a column. That doesn't seem right.

Reply:
Thanks Becky, I've filed this to track this and plan to close this PR: https://github.com/microsoft/AdaptiveCards/issues/4312